controller/destination/server.go

```diff
@@ -133,15 +131,11 @@ func buildResolversList(k8sDNSZone string, endpointsWatcher k8s.EndpointsWatcher
 		}
 	}
 
-	ipResolver := &echoIpV4Resolver{}
-	log.Infof("Adding ipv4 name resolver")
-
 	k8sResolver := &k8sResolver{
 		k8sDNSZoneLabels: k8sDNSZoneLabels,
 		endpointsWatcher: endpointsWatcher,
-		dnsWatcher:       dnsWatcher,
 	}
 	log.Infof("Adding k8s name resolver")
 
-	return []streamingDestinationResolver{ipResolver, k8sResolver}, nil
+	return []streamingDestinationResolver{k8sResolver}, nil
```

Comment:
Did you mean to remove the `echoIpV4Resolver` as part of this branch? That seems unrelated to removing external name support. I'm not sure what the proxy uses the ip resolver for, but functionally, if the path to be resolved is an ipv4 address, this resolver sends it back as an Add event. On master, it looks like this:

```
$ bin/go-run controller/script/destination-client -path 1.2.3.4:80
INFO[0000] Add:                                         
INFO[0000] metric_labels: map[]                         
INFO[0000] - 1.2.3.4:80 - map[]                         
INFO[0000]                                              
```

Whereas on this branch, I get:

```
$ bin/go-run controller/script/destination-client -path 1.2.3.4:80
FATA[0000] rpc error: code = Unknown desc = resolver [&{k8sDNSZoneLabels:[] endpointsWatcher:0xc4201b0420}] found error resolving host [1.2.3.4] port[80]: DNS name cannot only contain digits and hyphens: 1.2.3.4
```

Reply:
> Did you mean to remove the echoIpV4Resolver as part of this branch?

I should have removed it in an earlier branch. The proxy never sends IP addresses to the controller any more. I think even in 0.3.1 it was that way.

Reply:
Ok, in that case can you also remove the `echoIpV4Resolver` struct and associate methods and tests? I think that `controller/destination/resolver_test.go` can go away entirely.